helix-term/tests/test/commands.rs

```diff
@@ -25,6 +25,42 @@ async fn test_write_quit_fail() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn test_read_file() -> anyhow::Result<()> {
+    let mut file = tempfile::NamedTempFile::new()?;
+    let expected_content = String::from("some contents");
+    let output_file = helpers::temp_file_with_contents(&expected_content)?;
+    let mut command = String::new();
+    let cmd = format!(":r {:?}<ret><esc>:w<ret>", output_file.path());
+    command.push_str(&cmd);
+
+    println!(">>File: {:?}", file.path());
+    println!(">>Output: {:?}", output_file.path());
+    println!(
+        ">>input: {:?}",
+        std::fs::canonicalize(output_file.path()).unwrap()
+    );
```

Comment:
Are these for debugging? These lines can be removed

Reply:
Oops. Never happened.